webapp/channels/src/components/preparing_workspace/plugins.tsx

```diff
@@ -46,32 +45,19 @@ const Plugins = (props: Props) => {
         className += ' ' + props.className;
     }
 
-    let title = (
+    const title = (
         <FormattedMessage
-            id={'onboarding_wizard.cloud_plugins.title'}
-            defaultMessage='Welcome to Mattermost!'
+            id={'onboarding_wizard.self_hosted_plugins.title'}
```

Comment:
nit: we should rename this text id name to remove the self-hosted reference if it's now for both cases

```suggestion
            id={'onboarding_wizard.self_hosted_plugins.title'}
```